Controllers/RegisterController.cs

```diff
@@ -15,8 +13,8 @@ public RegisterController(MemberRegistration memberRegistration)
         _memberRegistration = memberRegistration;
     }
 
-    [HttpPost]
-    public async Task<IActionResult> Register(RegisterDTO userDetails)
+    [HttpPost("api/register")]
```

Comment:
This attribute can be changed back to default, and [Route("api/[controller]")] and [ApiController] Attributes should be added to the controller class